packages/server/routes/api/events.js

```diff
@@ -280,4 +281,58 @@ router.get(
   }
 );
 
+router.post(
+  "/:eventId/check-in",
+  AuthController.authenticate,
+  eventIdValidator,
+  userIdBodyValidator,
+  async (request, response) => {
+    const errors = validationResult(request);
+
+    if (!errors.isEmpty()) {
+      response.status(400).json({
+        status: "fail",
+        data: errors.array(),
+      });
+      return;
+    }
+
+    try {
+      const { status, userId } = request.body;
```

Comment:
This route is just for checking in users to events, so we should get rid of the `status` request body parameter since the status will always be `CheckedIn` when updating the event attendee status in the events controller.